src/main/scala/edu/berkeley/ce/rockslicing/Block.scala

```diff
@@ -63,6 +63,24 @@ object Block {
     }
   }
 
+  /** 
+    * Determines whether the input point is outside the input block
+    * @param point The point as a tuple
+    * @param block The block that the point is being checked agains
+    * @return True if the point is outside the block, false otherwise
+    */
+  private def pointOutsideBlock(point: (Double, Double, Double), faces: Seq[Face]): Boolean = {
+    val distanceChecks = faces map { face =>
```

Comment:
Getting there. Instead of `map` you can just use `exists` and flip the predicate.
```scala
faces exists { face =>
  ...
  ((pointVector dot faceNormal) - face.d) > NumericUtils.EPSILON
}